app/helpers/orchid/date_helper.rb

```diff
@@ -0,0 +1,75 @@
+module Orchid::DateHelper
+
+  def clear_dates_params
+    # duplicate parameters, remove date fields, and return
+    options = params.permit!.deep_dup
+    options.delete("date_from")
+    options.delete("date_to")
+    return options
+  end
+
+  # TODO could abstract this into a CDRH gem
+  def date_default(date_params, default_date)
+    y, m, d = date_params.map { |x| x.gsub(/\s+/, "") }
+    y = default_date[0] if y.blank?
+    m = default_date[1] if m.blank?
+    d = default_date[2] if d.blank?
+
+    # solr can't handle months and days without zero padding (01)
```

Comment:
whoops, missed the solr ref.  Elasticsearch is actually fine without padding, but I kind of like the idea of continuing to do it so that everything is uniform.  I'll change the comment